webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/VariableAndSectionSpec.scala

```diff
@@ -248,10 +250,10 @@ final case class TrackerVariableSpec(
   override type T = TrackerVariableSpec
   override type V = TrackerVariable
 
-  override val name:        String = TRACKINGKEY
-  override val description: String = "Variable which kept information about the policy"
-
-  override val checked: Boolean = false
+  override val name:         String         = TRACKINGKEY
+  override val description:  String         = "Variable which kept information about the policy"
+  override val variableName: Option[String] = None
```

Comment:
there's something I don't get here: we have : 
- name
- description
- variableName
- longDescription
- id

But we only have in yaml / metadata: 
- id
- name
- description
- documentation

Mapping from/to longdescription <-> documentation is clear ; but I don't get the remaining ones. 

I'm almost sure `name` actually contains `id` (always). And then should be named `id`. 
I'm not sure about what contains `id: Option[String]` nor name. 
And I think there is a difference between a "display name" which is perhaps a description ?